util/uvmdvgen/checklist.md.tpl

```diff
@@ -104,7 +104,6 @@ Review        | [REVIEW_SW_CSR][]       | Not Started |
 Review        | [REVIEW_SW_FATAL_ERR][] | Not Started |
 Review        | [REVIEW_SW_CHANGE][]    | Not Started |
 Review        | [REVIEW_SW_ERRATA][]    | Not Started |
-Security      | [SEC_CM_IMPLEMENTED][]  | Not Started |
```

Comment:
@msfschaffner @tjaychen believe this item is already covered in D2S line 83. Do you intend to put it here again? If so looks like we missed `[SEC_CM_IMPLEMENTED]: {{<relref "/doc/project/checklist.md#sec_cm_implemented" >}}` in the following section. 
Could you help confirm?

Reply:
yes we can remove this, thanks for catching that.
can you quickly check whether we also need to correct this in all checklist instances?

Reply:
Ofc! Just want to confirm with you before checking all the IPs ;)